.eslintrc.json

```diff
@@ -47,15 +43,10 @@
     "react/jsx-filename-extension": [2, { "extensions": [".js", ".jsx", ".ts", ".tsx"] }],
     "@typescript-eslint/ban-ts-comment": ["error", { "ts-ignore": "allow-with-description" }],
     "no-shadow": "off",
-    "@typescript-eslint/no-shadow": ["error"]
+    "@typescript-eslint/no-shadow": ["error"],
+    "@typescript-eslint/no-unused-expressions": "warn"
   },
   "overrides": [
-    {
-      "files": ["**/*.{ts,tsx}"],
-      "parser": "@typescript-eslint/parser",
-      "extends": ["plugin:@typescript-eslint/recommended"],
-      "plugins": ["jest", "react", "@typescript-eslint"]
-    },
```

Comment:
now it is a default one, airbnb-typescript applyes @typescript-eslint/eslint-plugin and @typescript-eslint/parser